ViewModel/FrequencyRelationships/GraphicalVM.cs

```diff
@@ -13,9 +13,12 @@ namespace HEC.FDA.ViewModel.FrequencyRelationships
     public class GraphicalVM : ComputeComponentVM
     {
         private int _equivalentRecordLength = 5;
+        private bool _useStage;
+        private bool _useFlow = true;
         private NamedAction _confidenceLimits;
-        public NamedAction ConfidenceLimits { get { return _confidenceLimits; } set { _confidenceLimits = value; NotifyPropertyChanged(); } }
 
+
+        public NamedAction ConfidenceLimits { get { return _confidenceLimits; } set { _confidenceLimits = value; NotifyPropertyChanged(); } }
```

Comment:
It is best to put getters and setters on different lines. This becomes helpful when needing to put breakpoints on them.